### PR TITLE
[LLDB] Xcode CustomSwift-Debug

### PR DESCRIPTION
**Pull request by weliveindetail**
Fix build configuration CustomSwift-Debug for xcodebuild

Broken since:
e25c6ecc10c85e3013b663410e8a280aca383e9f (Tue Jan 9 17:14:44 2018)
f0c3e96cd75c66753001b6285a3220464061be0f (Mon Mar 19 15:42:04 2018)

### CODE REVIEW

**Comment by weliveindetail** [comment]
@swift-ci test and merge

File: lldb.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -13455,130 +13450,159 @@
 			};
 			name = BuildAndIntegration;
 		};
-		94DF55CC1ADDBF240030B937 /* BuildAndIntegration */ = {
+		94DF55CC1ADDBF240030B937 /* CustomSwift-Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+ 				ARCHS = "$(ARCHS_STANDARD_64_BIT)";
+				"ARCHS[sdk=iphoneos*]" = arm64;
+				"ARCHS[sdk=macosx*]" = "$(ARCHS_STANDARD_64_BIT)";
```
Comment (fredriss) [inline lldb.xcodeproj/project.pbxproj:13458]: You can remove those 2 ARCHS[] lines

**Review by dcci — COMMENTED** [review]
Just FYI, Jason has a script to deterministically sort the pbxproj before committing (from a quick glance, it doesn't seem it should matter at all here, but running it doesn't hurt anyway)

File: lldb.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -13455,130 +13450,159 @@
...
+ 				LLDB_ZLIB_LDFLAGS = "-lz";
+ 				LLVM_BUILD_DIR = "$(LLDB_PATH_TO_LLVM_BUILD)";
+ 				LLVM_BUILD_DIRTREE = "$(PREBUILT_SWIFTLANG)/..";
+ 				LLVM_BUILD_DIR_ARCH = "llvm-macosx-$(CURRENT_ARCH)/";
+ 				"LLVM_BUILD_DIR_ARCH[sdk=appletvos*]" = "llvm-appletvos-$(CURRENT_ARCH)/";
+ 				"LLVM_BUILD_DIR_ARCH[sdk=iphoneos*]" = "llvm-iphoneos-$(CURRENT_ARCH)/";
+ 				"LLVM_BUILD_DIR_ARCH[sdk=watchos*]" = "llvm-watchos-$(CURRENT_ARCH)/";
+ 				LLVM_CONFIGURATION = DebugAssert;
+ 				LLVM_SOURCE_DIR = "$(SRCROOT)/llvm";
+ 				MACOSX_DEPLOYMENT_TARGET = 10.9;
```
Comment (fredriss) [inline lldb.xcodeproj/project.pbxproj:13547]: Remove the MACOSX_DEPLOYMENT_TARGET line too

**Review by fredriss — COMMENTED** [review]
This seems mostly fine to me, but we need to check that the BuildAndIntegration configuration still works. Jim and/or Jason might have a better idea about how to verify the change.

**Comment by weliveindetail** [comment]
Oh sorry, I didn't want to merge already, only test. Will revert that and make a new PR.

**Comment by fredriss** [comment]
It's fine, just make the tweaks I asked for

**Pull request merged**